src/test/java/uk/gov/justice/hmpps/casenotes/repository/OffenderCaseNoteRepositoryTest.java

```diff
@@ -198,6 +198,17 @@ public void findByModifiedDate() {
         assertThat(rows).extracting(OffenderCaseNote::getNoteText).contains(newNoteText).doesNotContain(oldNoteText);
     }
 
+    @Test
+    public void testGenerationOfEventId() {
+        final var note = repository.save(transientEntity());
+
+        TestTransaction.flagForCommit();
```

Comment:
the sequence value is written as a default, so have to commit and re-read from scratch to get the value